### ISSUE TITLE
xTimeZone tests fail on local machine if module is not deployed

### ISSUE THREAD

**Issue by KarolKaczmarek** [PowerShell/xTimeZone#11]
```
$timeZoneResource = Get-DscResource -Name xTimeZone
```
expects module to be deployed on the machine, but we don't do it (in appveyor we overwrite PSModulePath which does not solve the problem locally).

**Comment by KarolKaczmarek** [comment]
The side effect of that fix is that if user already had resources deployed on the machine, tests will fail as well with many errors similar to the one below:
```
ImportCimAndScriptKeywordsFromModule : A second CIM class definition for 'MSFT_xWebsite' was found while processing the schema file 
'C:\git\xDscResources-Designer\xWebAdministration\DSCResources\MSFT_xWebsite\MSFT_xWebsite.Schema.mof'. This class was already defined in the file 'C:\Program 
Files\WindowsPowerShell\Modules\xWebAdministration\DSCResources\MSFT_xWebsite\MSFT_xWebsite.Schema.mof'. Remove the redundant definition, and then try again.
At C:\windows\system32\windowspowershell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:3369 char:29
+ ...     $null = ImportCimAndScriptKeywordsFromModule -Module $mod -Resour ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], PSInvalidOperationException
    + FullyQualifiedErrorId : System.Management.Automation.PSInvalidOperationException,ImportCimAndScriptKeywordsFromModule
```

Not sure if we want to make tests robust when handling this scenario, but wanted to point it out..

**Comment by KarolKaczmarek** [comment]
instead of copying the module to Program Files, we should make sure we add the parent of the test folder to the PSModulePath, similarily as suggested here:
https://github.com/PowerShell/DscResource.Tests/issues/12

**Comment by PlagueHO** [comment]
This should be fixed when I make the changes for Issue #15.
